File: Java/268.java
__________________________________________________________________________________________________
sample 0 ms submission
class Solution {
    public int missingNumber(int[] nums) {
        int sum=0;
        for(int n: nums)
            sum += n;
        return nums.length*(nums.length+1)/2-sum;
        
    }
}
__________________________________________________________________________________________________
sample 36644 kb submission
class Solution {
    public int missingNumber(int[] nums) {
        
        int sum = 0;
        for(int n:nums)
        {
             sum+=n;
        } 
        return (nums.length * (nums.length+1)/2) - sum;
    }
}
__________________________________________________________________________________________________

File: Python3/268.py
__________________________________________________________________________________________________
sample 136 ms submission
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        
        s = sum(nums)
        
        n = len(nums)
        
        return n*(n+1)//2 - s
__________________________________________________________________________________________________
sample 13876 kb submission
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        return int(n*(n+1)/2 - sum(nums))
__________________________________________________________________________________________________
